client/src/components/FoodSeeker/SearchResults/AdvancedFilters/AdvancedFilters.js

```diff
@@ -0,0 +1,83 @@
+import React, { useCallback } from 'react';
+import PropTypes from "prop-types";
+import { Grid } from "@mui/material";
+import PantryButton from './FilterButtons/PantryButton';
+import MealButton from './FilterButtons/MealButton';
+import OpenNowButton from './FilterButtons/OpenNowButton';
+import MoreFiltersButton from './FilterButtons/MoreFiltersButton';
+import DaysButton from './FilterButtons/DaysButton';
+import * as analytics from "services/analytics";
+import useBreakpoints from 'hooks/useBreakpoints';
+import {
+  MEAL_PROGRAM_CATEGORY_ID,
+  FOOD_PANTRY_CATEGORY_ID,
+} from "constants/stakeholder";
+const AdvancedFilters = ({
+  toggleCategory,
+  categoryIds,
+}) => {
+  const { isMobile } = useBreakpoints();
+  const toggleMeal = useCallback(() => {
+    toggleCategory(MEAL_PROGRAM_CATEGORY_ID);
+    analytics.postEvent("toggleMealFilter", {});
+  }, [toggleCategory]);
+
+  const togglePantry = useCallback(() => {
+    toggleCategory(FOOD_PANTRY_CATEGORY_ID);
+    analytics.postEvent("togglePantryFilter", {});
+  }, [toggleCategory]);
+
+  const isMealSelected = categoryIds.indexOf(MEAL_PROGRAM_CATEGORY_ID) >= 0;
+  const isPantrySelected = categoryIds.indexOf(FOOD_PANTRY_CATEGORY_ID) >= 0;
+
+  return (
+    <Grid
+      container
+      wrap="nowrap"
+      display="inline-flex"
+      alignItems="flex-start"
+      sx={{
+        overflow: "auto",
+        gap: "0.5rem",
+        padding: isMobile ? "0 0 1rem 0.75rem" : "0 0 1rem 2.25rem",
+        marginTop: "0.75rem",
+        scrollbarWidth: "none",
+        '&::-webkit-scrollbar': {
+          display: 'none',
+        },
+        '&-ms-overflow-style:': {
+          display: 'none',
+        },
+      }}
+    >
+      <Grid item sx={{ whiteSpace: "nowrap" }}>
+        <PantryButton
+          toggleCategory={togglePantry}
+          isSelected={isPantrySelected}
+        />
```

Comment:
I think the new button components are not necessary. We can just use the `AdvancedFilterButton` directly or we can add a new button variant 🤔

Reply:
Fixed